openhands/core/config/utils.py

```diff
@@ -23,6 +23,19 @@
 load_dotenv()
 
 
+def create_llm_config(llm_config_dict: dict) -> LLMConfig:
+    """Create an LLMConfig object from a dictionary.
+
+    This function is used to create an LLMConfig object from a dictionary,
+    with the exception of the 'draft_editor' key, which is a nested LLMConfig object.
+    """
+    args = {k: v for k, v in llm_config_dict.items() if not isinstance(v, dict)}
+    if 'draft_editor' in llm_config_dict:
+        draft_editor_config = LLMConfig(**llm_config_dict['draft_editor'])
+        args['draft_editor'] = draft_editor_config
+    return LLMConfig(**args)
```

Comment:
It seems like it would be better for this to be in the `LLMConfig` class itself?

Reply:
good point, refactor it to a class method.